blocks/api/parser.js

```diff
@@ -123,13 +123,54 @@ export function isValidBlock( rawContent, blockType, attributes ) {
 
 	const isValid = ( actual === expected );
 
-	if ( ! isValid && 'development' === process.env.NODE_ENV ) {
-		// eslint-disable-next-line no-console
-		console.error(
-			'Invalid block parse\n' +
-				'\tExpected: ' + expected + '\n' +
-				'\tActual:   ' + actual
-		);
+	if ( ! isValid ) {
+		// NOTE: DO NOT COLLAPSE THESE CONDITIONS OR CONVERT COMMONJS TO ES2015
+		// IMPORTS. They are intentionally written as such because minification
+		// will drop this unreachable block in production environments.
+		if ( 'production' !== process.env.NODE_ENV ) {
+			const chalk = require( 'chalk' );
+			let message = 'Invalid block parse for ';
+
+			// Chalk automatically detects its environment to determine whether
+			// ANSI codes are supported. In these environments we provide a nicely
+			// colored error message for invalid parse.
+			//
+			// TODO: Teach browsers to ANSI
```

Comment:
... or teach `chalk` to output HTML.  ANSI escape sequences are quite convoluted, and it seems like what we really want is an intermediate representation that can be outputted in either form.